src/geojson.rs

```diff
@@ -451,6 +457,39 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_geojson_from_features() {
+        let features: Vec<Feature> = vec![
+            Value::Point(vec![0., 0., 0.]).into(),
+            Value::Point(vec![1., 1., 1.]).into(),
+        ];
+
+        let geojson: GeoJson = features.into();
+        assert_eq!(
+            geojson,
+            GeoJson::FeatureCollection(FeatureCollection {
+                features: vec![
+                    Feature {
```

Comment:
This is verbose. Worth just repeating the same `Value::Point(vec![0., 0., 0.]).into()` from above, since we're not trying to also test the conversion from `Value` to `Feature`?

Reply:
I have no strong feelings about it. I think it's fine for a test.